consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/CliqueProtocolSchedule.java

```diff
@@ -27,44 +35,33 @@
   public static ProtocolSchedule<CliqueContext> create(
       final GenesisConfigOptions config, final KeyPair nodeKeys) {
 
-    // Get Config Data
     final CliqueConfigOptions cliqueConfig = config.getCliqueConfigOptions();
-    final long epochLength = cliqueConfig.getEpochLength();
-    final long blockPeriod = cliqueConfig.getBlockPeriodSeconds();
-    final int chainId = config.getChainId().orElse(DEFAULT_CHAIN_ID);
-
-    final MutableProtocolSchedule<CliqueContext> protocolSchedule =
-        new MutableProtocolSchedule<>(chainId);
 
     // TODO(tmm) replace address with passed in node data (coming later)
```

Comment:
can you remove this todo? Appears its fulfilled.

Reply:
Done